requirements.md

```diff
@@ -30,13 +34,15 @@ See also: [SPC Scope](scope.md) for definitions and more information.
 ### Instrument Information
 
 * Enrollment must include display information for the associated instrument.
+
+Q: Should this be a 'must' if we're allowing for per-RP credentials (i.e. as opposed to requiring per-payment-instrument)? I could see it as an optional include in that case.
+
 * Because instrument display information is available to the relying party (e.g., provided by the relying party itself, the merchant, or some other party), it is not a requirement that this information be stored in the browser as part of the SPC Credential.
 
 ## Transaction Confirmation User Experience
 
 * Each browser must natively support a transaction confirmation user experience.
 * Although we anticipate that in most cases the browser will render the transaction confirmation user experience, the protocol must support rendering by other entities (e.g., the operating system or authenticator).
-* See [issue 48](https://github.com/w3c/secure-payment-confirmation/issues/48) on merchant information display.
```

Comment:
This is the same bullet as two-down, I think.

Reply:
Agreed. Removed the dup and closing here.